frontend/src/metabase/admin/settings/components/SettingsSlackForm.jsx

```diff
@@ -121,14 +121,6 @@ export default class SettingsSlackForm extends Component {
         [element.key]: MetabaseUtils.isEmpty(value) ? null : value,
       },
     });
-
```

Comment:
This file has been replaced by a new Slack form component and I think it can be totally removed. Maybe not necessary for this PR though.